MathLib/Integration/GaussLegendreTet.cpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "GaussLegendreTet.h"
+
+namespace MathLib
+{
+
+template <> const std::array<std::array<double, 3>, GaussLegendreTet<1>::NPoints>
+GaussLegendreTet<1>::X = {{{{1./3., 1./3., 1./3.}}}};
+template <> double const GaussLegendreTet<1>::W[1] = {1.0};
+
+const std::array<std::array<double, 3>, GaussLegendreTet<2>::NPoints>
+GaussLegendreTet<2>::X = {{ {{1./4., 1./4., 1./4.}},
+                            {{1./6., 1./6., 1./6.}},
+                            {{1./2., 1./6., 1./6.}},
+                            {{1./6., 1./2., 1./6.}},
+                            {{1./6., 1./6., 1./2.}} }};
+double const GaussLegendreTet<2>::W[5] = {-0.133333333333333, 0.075, 0.075, 0.075, 0.075};
+
+const std::array<std::array<double, 3>, GaussLegendreTet<3>::NPoints>
+GaussLegendreTet<3>::X = {{ {{1./2., 1./2., 1./2.}},
+                            {{0.09197107805272303, 0.09197107805272303, 0.09197107805272303}},
```

Comment:
We have unfortunately to wait until all of our visual studios get updated to the newest version. The current one does not support `constexpr`.